### PR TITLE
feat(api): camelcase interceptor

### PR DESCRIPTION
**Pull request by juanfran**

### CODE REVIEW

File: src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import { catchError, map } from 'rxjs/operators';
+import { Router } from '@angular/router';
+import { camelCase, snakeCase } from 'change-case';
+
+import { ConfigService } from '@/app/config.service';
+
+type TranformerInput = object | ArrayLike<unknown>;
+
+@Injectable()
+export class AuthInterceptorService implements HttpInterceptor {
```
Comment (bameda) [inline src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts:22]: Is there a typo in the class name?

File: src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        return throwError(err);
+      })
+    );
+  }
+
+  response(handler: HttpHandler, request: HttpRequest<any>) {
+    return handler.handle(request).pipe(
+      map((event) => {
+        if (event instanceof HttpResponse) {
+          return this.cameCaseResponseInterceptor(event);
```
Comment (Xaviju) [inline src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts:59]: typo `cameCaseResponseInterceptor` > `camelCaseResponseInterceptor`

Comment (juanfran) [reply src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts]: fixed!

File: src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts (hunk 1 above)
Comment (juanfran) [reply src/app/commons/api-rest-interceptor/api-rest-interceptor.service.ts]: yeah! fixed

**Pull request merged**